guarddog/analyzer/sourcecode/exfiltrate-sensitive-data.yml

```diff
@@ -31,6 +31,19 @@ rules:
           - metavariable-regex:
               metavariable: $ENVVAR
               regex: ([\"\'](AWS_ACCESS_KEY_ID|AWS_SECRET_ACCESS_KEY|AWS_SESSION_TOKEN)[\"\'])
+      - patterns:
+          - pattern-inside: |
+              $CONNECT = sqlite3.connect(...)
+              ...
+              $CURSOR = $CONNECT.cursor(...)
+              ...
+              $CURSOR.close(...)
+          - pattern: $CURSOR.execute($QUERY, ...)
+          - metavariable-pattern:
+              metavariable: $QUERY
+              patterns:
+                - pattern: "..."
+                - pattern-regex: (?i)select\s+[card_number_encrypted|encrypted_value|isHttpOnly|name|origin_url|password_value|username].*\s+from\s+[cookies|credit_cards|logins|moz_cookies|moz_formhistory|moz_logins]
```

Comment:
we can ditch the `(?i)select\s+[card_number_encrypted|encrypted_value|isHttpOnly|name|origin_url|password_value|username].*\s+from\s+`, let's say that if contains access any of those tables is interesting enough

also, i think you are using `[]` instead of `()`, it should be: `(cookies|credit_cards|logins|moz_cookies|moz_formhistory|moz_logins)`

isn't it?

Reply:
Sounds good, and yes, they should be `()` instead of `[]`